federatedscope/core/auxiliaries/splitter_builder.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import federatedscope.register as register
+
+logger = logging.getLogger(__name__)
+
+
+def get_splitter(config):
+    client_num = config.federate.client_num
+    args = eval('{' + config.data.splitter_args + '}')
```

Comment:
@xieyxclack @yxdyc @DavdGao This is also very tricky. Users have to read the code about splitters so that they know the args. Any better solution?